js/ui/grid_core/ui.grid_core.state_storing.js

```diff
@@ -246,25 +246,34 @@ module.exports = {
                     return this.callBase().concat(['stateLoaded']);
                 },
                 _refreshDataSource: function() {
-                    const that = this;
-                    const callBase = that.callBase;
-                    const stateStoringController = that.getController('stateStoring');
+                    const callBase = this.callBase;
+                    const stateStoringController = this.getController('stateStoring');
+                    const showErrorRow = (error) => {
+                        const errorHandling = this.getController('errorHandling');
+                        const errorMessage = error || 'Unknown error';
+                        errorHandling && errorHandling.renderErrorRow(errorMessage);
```

Comment:
It will be better to call this._handleLoadError(errorMessage) because the errorRowEnabled option and onDataErorOccured should works in this case..